src/osd/scheduler/OpScheduler.cc

```diff
@@ -33,15 +34,17 @@ OpSchedulerRef make_scheduler(
     type = &index_lookup[which];
   }
 
-  if (*type == "wpq" ) {
-    // default is 'wpq'
+  // Force the use of 'wpq' scheduler for filestore OSDs.
+  // The 'mclock_scheduler' is not supported for filestore OSDs.
+  if (*type == "wpq" || osd_objectstore == "filestore") {
```

Comment:
just thinking aloud, this won't allow users to even try mclock with filestore, but I guess that's fine